x-pack/libbeat/management/generate_test.go

```diff
@@ -0,0 +1,209 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package management
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/elastic/elastic-agent-client/v7/pkg/client"
+	"github.com/elastic/elastic-agent-client/v7/pkg/proto"
+	"github.com/elastic/elastic-agent-libs/mapstr"
+)
+
+func TestMBGenerate(t *testing.T) {
+	// following examples are built from this
+	// 	raw := `
```

Comment:
Can you explain what the _this_ is here (an agent policy)?

Reply:
Yah, agent policy I was using as an example when writing tests. Probably don't need it anymore...